pkg/updater/gcs.go

```diff
@@ -390,12 +391,17 @@ func convertResult(log logrus.FieldLogger, nameCfg nameConfig, id string, header
 
 	emailAddressesInterface, ok := result.finished.Finished.Metadata[EmailListKey]
```

Comment:
There are strings inside the `[]interface{}`, so you can do this:

```golang
var items []interface{}
for _, item := range items {
  s, ok := item.(string)
}
```

https://play.golang.org/p/vLuYFf_mUpB

Reply:
This should call one of the Metadata methods, rather than accessing the map.